lib/translators.js

```diff
@@ -0,0 +1,48 @@
+/*!
+ * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
+ */
+import {
+  ECDSA_2019_SECP_256_KEY_TYPE,
+  ECDSA_2019_SECP_384_KEY_TYPE,
+  ECDSA_2019_SECP_521_KEY_TYPE,
+  ECDSA_2019_SUITE_CONTEXT_V1_URL,
+  MULTIKEY_CONTEXT_V1_URL
+} from './constants.js';
+
+// converts key pair to Multikey format
+export async function toMultikey({keyPair}) {
+  const validEcdsaTypes = [
+    ECDSA_2019_SECP_256_KEY_TYPE,
+    ECDSA_2019_SECP_384_KEY_TYPE,
+    ECDSA_2019_SECP_521_KEY_TYPE
+  ];
+  if(!validEcdsaTypes.includes(keyPair.type)) {
```

Comment:
@gannan08 Is this check necessary/proper? I found these key type constants in the ECDSA 2019 Data Integrity spec (e.g., [`EcdsaSecp256r1VerificationKey2019`](https://w3c-ccg.github.io/di-ecdsa-2019/#ecdsasecp256r1verificationkey2019)) and I saw a similar check in the ED25519 Multikey library [here](https://github.com/digitalbazaar/ed25519-multikey/blob/main/lib/keyPairTranslator.js#L8). However, this may not be a necessary check for this special, especially since these constants that are defined in the spec are for SECP keys and may not even be necessary with the `Multikey` type.